skynet/agent.py

```diff
@@ -1,15 +1,23 @@
 import asyncio
 import math
 
+from fastapi import FastAPI, Request
+from pydantic import BaseModel
+
 from skynet.env import whisper_max_connections
 
 from skynet.logs import get_logger
-from skynet.modules.monitoring import CONNECTIONS_METRIC
+from skynet.modules.monitoring import CONNECTIONS_METRIC, TRANSCRIBE_GRACEFUL_SHUTDOWN, TRANSCRIBE_STRESS_LEVEL_METRIC
 
 log = get_logger(__name__)
+app = FastAPI()
+
+STATE_FILE = '/tmp/streaming-whisper-connections-state'
+haproxy_state = 'ready'
+haproxy_check_state_task = None
 
 
-async def calc_percentage():
+async def get_haproxy_percentage():
```

Comment:
Now that I look at this, there is nothing async inside this function, so why the async ?